lib/DTLZ/benchmark.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+from deephyper_benchmark import *
+
+DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+class DTLZ_lib(Benchmark):
```

Comment:
For class's names follow camel cases: "Start each word with a capital letter. Do not separate words with underscores. This style is called camel case or pascal case." No underscore, `DTLZBenchmark` maybe.

Reply:
yeah, mainly for consistency see `MPIDistributedBO`, `LCModelStopper`.